cobigen/cobigen-core/src/main/java/com/devonfw/cobigen/impl/util/ConfigurationFinder.java

```diff
@@ -196,7 +206,7 @@ private static URI findTemplates(Path home) {
     if (Files.exists(templatesPath)) {
       Path jarFilePath = TemplatesJarUtil.getJarFile(false, templatesPath);
       if (jarFilePath != null && Files.exists(jarFilePath)) {
-        return jarFilePath.toUri();
+        return templatesPath.toUri();
```

Comment:
I don't get this change, why should that be correct?

Reply:
I thought this will cause the bug I was trying to fix with this PR. But I have looked into it again and the javadoc of the method says it can also return a template artifact, I have overlooked that. The bug is causes by the TemplateAdapterImpl and should be fixed there.